### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by nschoot** [kohsuke/winsw#898] [General]
Looking at the activities, the project looks abandoned. No pull requests are accepted. Is there a reliable fork of this project?

**Comment by offa** [comment]
That's a question for the current maintainers @oleg-nenashev @nxtn

**Comment by xzycn** [comment]
There are now so many bugs listed :(

**Comment by oleg-nenashev** [comment]
Hello. As it is stated in my GitHub profile, I am not actively available on GitHub at the moment, beyond my direct work responsibilities. There is a war going on in Ukraine, and I am busy with volunteering and with some other personal efforts. I would be happy to help other maintainers to step in, but at the moment I cannot dedicate enough time

**Reply by nonnex** [reply]
Oh, 
well that explains a lot and is a hell of a reason Oleg, did'nt knew that. 
Very big² Shiat + mucho F**k.

Take care, be save bro!
PS: Where're You atm?

**Reply by oleg-nenashev** [reply]
I am in Switzerland. And, to avoid any wrong conclusions, I'm not a Ukrainian citizen/resident and not in any kind of direct danger. I am a Russian citizen, who left the country almost a decade ago, though my decision was greatly influenced by the first invasion in 2014.

What I meant is that I prioritized volunteering, and not only the humanitarian efforts, over most of my open source maintenance work. Of course, not on the Russian side. I have some minor inconveniences here and there, including limited travel capacity or employability in the areas of my direct interest, but nothing that would prevent me from maintaining this or other components, apart from my own decisions and financial responsibilities.

Still, I hope that users of WinSW understand my decisions. The project, while definitely in limbo, remains open to all kinds of contributions, remaining and I remain open to supporting anyone who wants to contribute or become a maintainer.

**Comment by oleg-nenashev** [comment]
Pinned the issue

**Comment by nonnex** [comment]
Good question.

Despite the last commits are some way back in time, I still have some hope that this thing is not going to be a Code Zombie and lost in the bins.

Because this WinSW Git is really the better "nssm" replacement, at least I know of, and I have learned to love it!
So I have no other choice of yelling: "No!" regarding a possible project death.

We **must** keep this alternative to nssm somehow alive and maintained!

Last resort: Fork... someone?

PS: I just don't want do go back to "sc queries" etc... seriously.

**Comment by oleg-nenashev** [comment]
It is not about forking, all contributions are still welcome though of
course there would be a limited number of them given the current state of
the branches. Once there is a capable computer willing to take ownership of
this component, I will be happy to hand it over without forking.

I still plan to do something to make the situation better and to make it
again contributable (restoring CI, docs, etc.), and actually over the past
months I had had conversations with two companies that were interested in
contributing some engineering time. While we had some discussions, they
haven't resulted in contributions yet.

**Comment by akoppa** [comment]
And who is Kohsuke Kawaguchi?

**Reply by oleg-nenashev** [reply]
> And who is Kohsuke Kawaguchi?

There is even a Wikipedia article about @kohsuke - https://en.wikipedia.org/wiki/Kohsuke_Kawaguchi
